app/(private-pages)/admin-dashboard/package/_components/edit-package-page.tsx

```diff
@@ -0,0 +1,13 @@
+'use client'
+import { useGetAllActivePackagesById } from "@/api/services/packages";
+import SingularPackageToEdit from "./singular-package-to-edit";
+import EditPackagePageSkeleton from "./edit -package-page-skeleton";
+import { Package } from "@/constants/types";
+
+export default function EditPackagePage ({ id }: { id:string}) {
+    const { data: packages, isLoading } = useGetAllActivePackagesById(id);
```

Comment:
this is not done properly. you are supposed to use the usegetpackagebyid hook and not useGetAllActivePackagesById(id)